README.md

```diff
@@ -1,5 +1,5 @@
-Auth0 for Mautic
-================
+Auth0 for Mautic 3
```

Comment:
Please do not harden the version number here as this plugin is in older versions still available for versions prior to Mautic 3.